src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -103,6 +109,58 @@ class IpexCommunicationService extends AgentService {
     });
   }
 
+  async offerAcdc(notificationId: string, acdc: any) {
+    const keriNoti = await this.getNotificationRecordById(notificationId);
+    const msgSaid = keriNoti.a.d as string;
+    const msg = await this.signifyClient.exchanges().get(msgSaid);
```

Comment:
Do we already have the message said available from the notification when it comes into the AppWrapper, or we must call `getNotificationRecordById` for sure?

Reply:
Hmm.. I saw that, I don't know why I did it before. Let me fix